recipe/patches/0002-get_xticks-returns-list.patch

```diff
@@ -0,0 +1,25 @@
+From 3b3a99c73bbf9ffd66780ce05f07bacdacb3deab Mon Sep 17 00:00:00 2001
+From: Lorenzo Pirritano <lpirritano@anaconda.com>
+Date: Tue, 3 Sep 2024 16:34:17 +0200
+Subject: [PATCH 2/2] get_xticks returns list
+
+---
+ ds_utils/preprocess.py | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/ds_utils/preprocess.py b/ds_utils/preprocess.py
+index 97bdeff..0879272 100644
+--- a/ds_utils/preprocess.py
++++ b/ds_utils/preprocess.py
+@@ -49,7 +49,7 @@ def visualize_feature(series: pd.Series, remove_na: bool = False, *, ax: Optiona
+         ax.set_title(f"{feature_series.name} ({feature_series.dtype})")
+         ax.set_xlabel("")
+ 
+-    ticks_loc = ax.get_xticks().tolist()
++    ticks_loc = ax.get_xticks()

```

Comment:
There are other occurrences of this in this file: https://github.com/idanmoradarthas/DataScienceUtils/blob/1.7.2/ds_utils/preprocess.py (L52, L241, L256)
matplotlib https://matplotlib.org/3.8.4/api/_as_gen/matplotlib.axes.Axes.get_xticks.html changed return type from numpy array of tick locations to array of tick locations in 3.8. Given the package declares a min of matplotlib 3.8, it would make sense to correct the other occurrences and submit a PR upstream.

Reply:
that's neat, thanks for having a look at this. Good catch. I now wonder if any other of the `.tolist()` in the same file has the same problem. I will contact the upstream author, I already had a PR merged for the dependencies.

Reply:
https://github.com/idanmoradarthas/DataScienceUtils/issues/35

Reply:
I will need to postpone this to Monday

Reply:
done